fakeData.js

```diff
@@ -0,0 +1,46 @@
+const faker = require('faker');
+let attractions = [];
+
+function getRandomNumBetween(min, max) {
+    return Math.floor(Math.random() * (max - min) + min);
+}
+
+for (let attraction = 0; attraction < 200; attraction++){
+    let attractionQuestionsAndAnswers = {};
+
+    attractionQuestionsAndAnswers.id = attraction;
```

Comment:
Miscommunication, sorry. I meant ```attractionQuestionsAndAnswers``` should just be ```attraction```. The iterator should probably still be ```i```.